src/content/developers/docs/data-structures/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Data structures
+description: An overview of the fundamental Ethereum data structures.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+Ethereum creates, stores and transfers large volumes of data. This data must get formatted in standardized and memory-efficient ways to allow anyone to [run a node](/run-a-node/) on relatively modest consumer-grade hardware. To achieve this, several specific data structures are used on the Ethereum stack.
+
+## Prerequisites {#prerequisites}
+
+You should understand the fundamentals of Ethereum and [client software]. Familiarity with the networking layer and [the Ethereum whitepaper](/whitepaper/) is recommended.
+
+## Data structures {#data-structures}
+
+### Patricia merkle tries {#patricia-merkle-tries}
+
+Patricia Merkle Tries are structures that encode key-value pairs into a deterministic trie. These are used extensively across Ethereum's execution layer. 
+
+[More on Patricia Merkle Tries](developers/docs/data-structures/patricia-merkle-trie)
+
+### Recursive Length Prefix {#recursive-length-prefix}
```

Comment:
I'm not sure data structure is the best term to describe RLP and SSZ encoding. Maybe they are in some abstract sense but what about extending the title to something like "data structures and formats/encoding/serialization" or something of the sort

Reply:
updated directory name and page title